### ISSUE TITLE
Weird memory requirements in console vs. Jupyter notebook

### ISSUE THREAD

**Issue by domaala** [pytorch/text#307]
I've been running code that loads pre-trained FastText word embeddings in the Jupyter notebook (JupyterLab, to be precise) without any major issues for days. However, after I decided to extract model training into a separate Python script to be run for a longer period of time, I experienced enormous memory requirements. I hit the same wall with Jupyter console as well.

The issue of a memory leak of sorts is directly mentioned in #298 .  

tl;dr Jupyter notebooks handle loading big embeddings in an expected manner, while Jupyter console and regular Python script do not. There could be some problems with either PyTorch of TorchText under different environments.

**Comment by zhangguanheng66** [comment]
@domaala do you still have this issue under current versions of Pytorch and torchtext? Feel free to reopen it.
